x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.tsx

```diff
@@ -120,34 +122,30 @@ const RuleActionsOverflowComponent = ({
           ]
         : [],
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [rule, userHasNoPermissions]
+    [rule, userHasPermissions]
```

Comment:
Above the `eslint-disable-next-line react-hooks/exhaustive-deps` to me is legacy and big big problem. For a long time we didn't have the linter rule turned on, then we turned it on and just kind of disabled it everywhere. If possible, when I touch code and see it in a file I always try to see if what is preventing me from removing it.

Is it possible to remove the disablement of the liter rule and add the required deps?

Reply:
Yeah, I don't see a strong reason not to include all deps there. Will do that 👍